packages/dapp/src/views/editBasicProfile/index.tsx

```diff
@@ -178,6 +178,7 @@ const EditBasicProfile = ({
             placeholder="anon"
             borderColor="purple.500"
             {...register("name", {
+              required: true,
```

Comment:
We was allowing empty names and then anonymous was displayed. I'm not sure that this was a feature or a bug. Maybe we can leave it as optional and then we can as @QEDK about it.